modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/index.tsx

```diff
@@ -206,6 +208,15 @@ const Layout: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 
 		const parentWindow = Liferay.Util.getOpener();
 
+		if (objectLayout.name[defaultLanguageId] === '') {
```

Comment:
Use `invalidateRequired()` in order to enforce same rules followed in the `InfoScreen.tsx`
```suggestion
		if (invalidateRequired(objectLayout.name[defaultLanguageId])) {
```